pipelines/MetricsRuntime.yaml

```diff
@@ -24,54 +24,38 @@ tasks:
       connections.metric: CalibrateTime
       connections.labelName: calibrate
       target: calibrate.run
-  timing_imageDifference:
+  timing_subtractImages:
     class: lsst.verify.tasks.commonMetrics.TimingMetricTask
     config:
-      connections.package: pipe_tasks
+      connections.package: ip_diffim
       connections.metric: ImageDifferenceTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference.run
-  timing_imageDifference_astrometer:
-    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
-    config:
-      connections.package: meas_astrom
-      connections.metric: AstrometryTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference:astrometer.loadAndMatch
-  timing_imageDifference_register:
-    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
-    config:
-      connections.package: pipe_tasks
-      connections.metric: RegisterImageTime
-      connections.labelName: imageDifference
-      metadataDimensions: [instrument, visit, detector, skymap]
-      target: imageDifference:register.run
-  timing_imageDifference_subtract:
+      connections.labelName: subtractImages
+      metadataDimensions: [instrument, visit, detector]
+      target: subtractImages.run
     class: lsst.verify.tasks.commonMetrics.TimingMetricTask
+  timing_detectAndMeasure:
     config:
```

Comment:
There's no `class` key here, which breaks the pipeline. Please fix and confirm that the pipeline works.